Source/WebKit/WPEPlatform/wpe/drm/WPEScreenDRM.cpp

```diff
@@ -41,6 +42,10 @@ struct _WPEScreenDRMPrivate {
     std::unique_ptr<WPE::DRM::Crtc> crtc;
     drmModeModeInfo mode;
     GRefPtr<WPEScreenSyncObserver> syncObserver;
+    struct {
+        uint32_t bufferID;
+        uint32_t frameBufferID;
+    } dumb;
```

Comment:
```suggestion
    } dumb;
```

I think the term `dummy` is better

Reply:
The "dumb buffer" is actually a DRM term.

Reply:
Dumb buffer is a DRM name, but I agree we could use dummy instead.

Reply:
:upside_down_face: the case where not knowing helps :sweat_smile: